inc/taxonomies.php

```diff
@@ -62,13 +62,13 @@ function largo_custom_taxonomies() {
 
 	$termsDefinitions = array(
 		array(
-			'name' => __('Sidebar Featured Widget', 'largo'),
-			'description' => __('If you are using the Featured Posts widget in a sidebar, add this label to posts to determine which to display in the widget.', 'largo'),
+			'name' => __('Sidebar Featured', 'largo'),
```

Comment:
see note above, i think it's probably ok to just not automatically create these by default